dnsdist: build failure with gcc-15: ext/.../json11.cpp: error: uint8_t does not name a type

- [x] This is not a support question, I have read [about opensource](https://www.powerdns.com/opensource.html) and will send support questions to the IRC channel, [Github Discussions](https://github.com/PowerDNS/pdns/discussions/) or the mailing list.
- [x] I have read and understood the ['out in the open' support policy](https://blog.powerdns.com/2016/01/18/open-source-support-out-in-the-open/)

<!-- Tell us what is issue is about -->
 - Program: dnsdist
 - Issue type: Bug report

### Short description
Gentoo is test-driving early gcc-15 snapshots and found a (trivial) problem:
https://bugs.gentoo.org/937628 (see below for the fix :rocket:)

### Environment
 - Operating system: Gentoo Linux
 - Software version: any
 - Software source: existing package for 1.9.6

### Steps to reproduce
Issue can be reproduced by building dnsdist (probably also pdns?) with gcc-15.

### Expected behaviour
Successful build

### Actual behaviour
```
ext/json11/json11.cpp: In function void json11::dump(const std::string&, std::string&):
ext/json11/json11.cpp:96:32: error: uint8_t does not name a type
   96 |         } else if (static_cast<uint8_t>(ch) <= 0x1f) {
      |                                ^~~~~~~
```
### Other information
The bug is trivial - just a missing include. The fix for the json11 package in Gentoo can be found [here](
https://gitweb.gentoo.org/repo/gentoo.git/commit/?id=79d5a2baf84f5a9086cf49095928bc9fb31cfb82).
Since json11 is archived and no longer under development we do not expect this to be fixed upstream.

--- a/ext/json11/json11.cpp
+++ b/ext/json11/json11.cpp
@@ -22,6 +22,7 @@
 #include "json11.hpp"
 #include <cassert>
 #include <cmath>
+#include <cstdint>
 #include <cstdlib>
 #include <cstdio>
 #include <limits>

